uploading/models/user_groa_tables/ratings.js

```diff
@@ -0,0 +1,27 @@
+const db = require("../../../database/dbConfig.js");
+
+module.exports = {
+  addRating,
+  getRatingById
+};
+
+/**
+ * Inserts a rating to the database for the Groa site
+ * @param {object} rating - date, name, year, rating, user_id
+ * @returns {ratedMovie}
+ */
+async function addRating(rating) {
+  await db("user_groa_ratings").insert(rating);
+  return { message: "Success!" };
```

Comment:
In the router it looks as though we are passing the rating back to the user, but here we're not. Do we want to?

Elsewise, this should be an upsert and check if the rating exists in the table before putting it in here, unless the database has constraints on the table to only allow one movie id per user id.

The above point leads me to wonder if we don't want all user data in one table? If you already reviewed something on letterboxd, or imdb that won't show up here, allowing duplicate, triplicate ratings..

Reply:
^^^ I'll go through and clean up before I make it ready to review this morning. And we should condense the tables.

Reply:
Upsert is now functional, as is returning the rating after successful addition. Will condense the tables next.